src/Time/Units.hs

```diff
@@ -244,15 +244,36 @@ fortnight = time
 -- Functional
 ----------------------------------------------------------------------------
 
--- | Converts from one time unit to another time unit.
+{- | Converts from one time unit to another time unit.
+
+>>> convertUnit @HourUnit (120 :: Minute)
+2h
+
+>>> convertUnit @SecondUnit (ms 7)
+7/1000s
+
+>>> convertUnit @WeekUnit (Time @DayUnit 45)
+45/7w
```

Comment:
Can we display this as `6+3/7w`?

Reply:
Ah, I see, there is an issue about it already https://github.com/serokell/o-clock/issues/19